content/en/getting_started/integrations/_index.md

```diff
@@ -83,7 +84,9 @@ instances:
 
 ### Tagging
 
-Tagging is a key part of filtering and aggregating the data coming into Datadog across many sources. You can assign tags in configuration files, in environment variables, in the UI, the API, and in DogStatsD. If you define tags in the `datadog.yaml` file, the tags are applied to all of your integrations data. Once you've defined the tag in `datadog.yaml`, all new integrations inherit it. If you use a tag environment variable, it applies to all integrations. If you define tags in the corresponding integrations configuration file, it only applies to that specific integration. If you use tags in containers, it applies only to that container. For more information about tagging, see [Getting started with tags][25].
+Tagging is a key part of filtering and aggregating the data coming into Datadog across many sources. For more information about tagging, see [Getting started with tags][25].
```

Comment:
I cannot find information about the `service` config option in integrations creating the service tag. This is the preferred way for traditional bare metal agent setups that are monitoring separate, independant, systems.

Reply:
Good point, we should add a more specific call out for setting `service`.

`env` would likely be configured via the agent. `version` _could_ be set but then folks would need to update their config + restart the agent every time they change the tag.

Reply:
Here are the PRs btw:

- https://github.com/DataDog/datadog-agent/pull/4877
- https://github.com/DataDog/datadog-agent/pull/4895
- https://github.com/DataDog/jmxfetch/pull/269